docs/learn/using-zeitgeist-markets.md

```diff
@@ -134,148 +53,100 @@ the losing outcome tokens (see [Resolving Markets and Redeeming Tokens]).
-:::important
-The market creator can specify _any_ address as oracle, but also provides the
-stake for the oracle. If the market creator specifies an unwitting oracle (by
-mistake or with malicious intent), the market creator will lose their stake and
-the oracle will go unpunished.
-:::
+### Creation Type: Permissionless & Advised Markets
 
-As mentioned earlier markets have no liquidity pool by default. The market
-creator can choose to deploy the liquidity pool during market creation or create
-the market without a liquidity pool, hoping that someone else will deploy a pool
-for the market.
+Th Zeitgeist network allow entirely _permissionless_ creation of market. Anyone
+can create a new market, provided they place a _validity bond_ in ZTG. The
+purpose of the bond is to ensure that the market obeys the rules put in place by
+the Zeitgeist team. It is returned to the market creator after the market is
+resolved. A permissionless market that violates the [market creation rules] may
+be _deleted without warning_, destroying all associated funds.
 
-<!-- prettier-ignore -->
-:::important
-The market creator will sign three transactions when deploying a liquidity pool
-during market creation: Creating the market and bonding the stake for the
-oracle, minting the outcome tokens, and joining the liquidity pool.
-:::
+If the user does not wish to place the validity bond, they may instead create
+the market as _advised_ and only place a smaller _advisory bond_. The [Advisory
+Committee] then decides if the market is valid. Until this decision is made, no
+liquidity pool may be deployed for the market and no full sets may be bought or
+sold.
 
-### During Market Hours
+If the market is deemed valid according to the [market creation rules], the
+advisory bond is returned to the creator and liquidity pools may be deployed.
+If, on the other hand, the market violated the rules or is invalid, the advisory
+bond is slashed or the user is asked to change the market before it is approved.
+Should, for any reason, the committee not reach a decision until the market
+ends, the advisory bond and oracle bond are both returned to the creator.
 
-The market opens immediately after it is created. If no liquidity pool was
-deployed, trading as described in [Trading on Zeitgeist] is impossible, but
-users may still mint/burn full sets of tokens.
+### Market Lifecycle
 
-The market remains open until the end date is reached. The market will then
-become _inactive_ and trading will no longer be possible.
+The market opens when the starting block/timestamp is reached. If a liquidity
+pool is deployed for the market during market hours or before, users can swap
+assets and may provide liquidity to the pool.
 
-### After Hours: Reporting an Outcome
+The market closes when the ending block/timestamp is reached. This means that
+swapping assets and providing liquidity is no longer allowed, but users are
+allowed to remove their liquidity now that it is no longer user.
```

Comment:
In CPMM LPs the reasoning was that pool funds can be withdrawn at any point, right? I think this is still the case for neo-swaps, right? Or not for the first version, but whenever we allow multiple LPs.

Reply:
Actually, in CPMM they can't withdraw all their funds while the market is still open. With LMSR, that's allowed.